src/app/video-courses/highlight-course.directive.ts

```diff
@@ -0,0 +1,37 @@
+import { Directive, ElementRef, Input, OnInit } from '@angular/core';
+
+@Directive({
+  selector: '[appHighlightCourse]'
+})
+export class HighlightCourseDirective implements OnInit {
+  @Input('appHighlightCourse') courseDate: Date;
+
+  private static isFreshCourse(courseDate: Date, currentDate: Date) {
+    const millisecondsInDay = 1000 * 60 * 60 * 24;
```

Comment:
Move to static prop, not calculate on each static method invocation.

Reply:
Done